todaythediary/src/components/todo/Form.js

```diff
@@ -0,0 +1,34 @@
+import React,{useState} from 'react';
+import { BsPlusCircleFill } from 'react-icons/bs';
+
+const Form = ({inputText,setInputText, todos, setTodos}) =>{
+
+    const inputTextHandler = (e) =>{
+        setInputText(e.target.value);
+    };
+    const [counter,setCounter] = useState(0);
+    const incrementer = ()=>{
```

Comment:
Renaming `counter` to `count`, `setCounter` to `setCounter` and `incrementer` to `incrementCount` would be better

Reply:
Thank You.
I will do that soon.